workflow-dispatch/README.md

```diff
@@ -0,0 +1,19 @@
+# Workflow dispatch
+
+Trigger a workflow run with the workflow_dispatch event.
+
+See [action.yml](./action.yml) for the list of `inputs` and `outputs`.
+
+## Example usage
+
+```yaml
+steps:
+  - uses: actions/checkout@v2
+  - name: Trigger my workflow
+    uses: agendrix/actions/workflow-dispatch@master
+    id: trigger-workflow
+    with:
+      token: ${{ secrets.MY_PAC }}
```

Comment:
Can you change the secret's name? For my `MY_PAC` does not ring any bell. Maybe be more specific.

Reply:
Sure no problem, PAC stands for Personal Access Token but maybe it is not super clear 😅

Reply:
Right. So then :D
```suggestion
      token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```